static/app/components/sidebar/sidebarAccordion.tsx

```diff
@@ -25,9 +25,14 @@ import SidebarItem, {isItemActive} from './sidebarItem';
 
 type SidebarAccordionProps = SidebarItemProps & {
   children?: React.ReactNode;
+  initiallyExpanded?: boolean;
```

Comment:
what about `collapsedByDefault`

Reply:
@evanpurkhiser you tell me! I went with `initiallyExpanded` to match how it's stored and referenced. Otherwise, you get:

```tsx
  const [expanded, setExpanded] = useLocalStorageState(
    `sidebar-accordion-${id}:expanded`,
    !collapsedByDefault
  );
```